src/app/components/SearchModal/CurrencyList.tsx

```diff
@@ -111,6 +130,9 @@ export default function CurrencyList({
             setRemoveToken(currency as Token);
             showRemoveView();
           }}
+          isFocused={index === focusIndex}
```

Comment:
Instead of handling index in child components, handle here.
`onFocus = { () => setFocusIndex(index) }`